src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < shortTerm) {
+    return days * basePrice;
+  } else if (days < longTerm) {
```

Comment:
Ensure that the condition `days < longTerm` correctly reflects the task requirements for applying the small discount. If the task specifies different conditions for discounts, adjust this logic accordingly.